DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,10 @@ public function getConfigTreeBuilder()
                             ->scalarNode('protocol')->defaultValue('Thrift\Protocol\TBinaryProtocolAccelerated')->end()
                             ->scalarNode('transport')->defaultValue('Thrift\Transport\TBufferedTransport')->end()
                             ->booleanNode('server')->defaultFalse()->end()
+                            ->arrayNode('includeDirs')
+                                ->prototype('scalar')
```

Comment:
Maybe it's better to define includeDirs directy as a prototype of array, to avoid casting in the setIncludeDirs method